### PR TITLE
Feature/arima improvements

### PR DESCRIPTION
**Pull request by StanislasGuinel**
Make AutoARIMA run Blas library with a single thread
On daily temperature, training time decreases from 230s to 160s when using a single thread (and with less CPU usage)

### CODE REVIEW

File: python-lib/gluonts_forecasts/custom_models/autoarima.py [hunk 1]
```diff
@@ -102,7 +103,10 @@ def train(self, training_data, validation_data=None):
             kwargs = self._set_seasonality(self.kwargs, item[TIMESERIES_KEYS.TARGET])
             external_features = self._set_external_features(kwargs, item)
 
-            model = pm.auto_arima(item[TIMESERIES_KEYS.TARGET], X=external_features, **kwargs)
+            with threadpool_limits(limits=1, user_api='blas'):
```
Comment (alexcombessie) [inline python-lib/gluonts_forecasts/custom_models/autoarima.py:106]: Are we certain the user_api is blas only?
Can we make limits a parameter of the model, with a default at 1?

**Comment by alexcombessie** [comment]
[ch60072](https://app.clubhouse.io/dataiku/story/60072/deactivate-autoarima-by-default)

**Comment by alexcombessie** [comment]
... and deactivate ARIMA by default in the UI and in the AutoML modes

File: python-lib/gluonts_forecasts/custom_models/autoarima.py (hunk 1 above)
Comment (StanislasGuinel) [reply python-lib/gluonts_forecasts/custom_models/autoarima.py]: 964894d

**Review by alexcombessie — APPROVED** [review]

**Pull request merged**